rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/HistoryInstances.scala

```diff
@@ -0,0 +1,16 @@
+package coop.rchain.rspace.nextgenrspace.history
+
+import cats.effect.Sync
+import cats.implicits._
+import cats.{Applicative, FlatMap}
+import coop.rchain.rspace.Blake2b256Hash
+import coop.rchain.rspace.nextgenrspace.history.History._
+
+object HistoryInstances {
+  def simplistic[F[_]: Sync](
```

Comment:
This name is a bit vague. How is this instance `simplistic`?

Reply:
will replace with `noMerging`